gulpfile.js

```diff
@@ -1,9 +1,7 @@
-﻿"use strict"
```

Comment:
any reason you are removing this use strict? In js files it actually serves a function.

Reply:
lgtm complained about it and doesn't look like it was doing anything

Reply:
do you have a link to it complaining? it shouldn't. It is always recommended to use this for js files. (and typescript, but the ts compiler adds that for us). https://www.w3schools.com/js/js_strict.asp